src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -764,6 +764,10 @@ internal override SyntaxNode AsInterfaceMember(SyntaxNode m)
                 Accessibility acc;
                 DeclarationModifiers modifiers;
 
+                // return any nested member "as is" without any additional changes
+                if (member is BaseTypeDeclarationSyntax)
+                    return member;
```

Comment:
Not sure if it's more safe to `return member` in the default case of switch below. @CyrusNajmabadi What do you think?

Reply:
We might have an incomplete member for example.

Reply:
i think this is ok.  IncompleteMember will likely need to be handled differently.